kernelci/build.py

```diff
@@ -756,7 +756,7 @@ def _check_min_kver(self, major, minor):
         m = self.KVER_RE.match(kver)
         if m and len(m.groups()) == 2:
             k_major, k_minor = (int(g) for g in m.groups())
-            return k_major >= major and k_minor >= minor
+            return k_major > major or (k_major == major and k_minor >= minor)
```

Comment:
Maybe we can use tuple comparison?

```suggestion
            return (k_major, k_minor) >= (major, minor)
```

Reply:
Ah yes, good point.  Will update it